README.md

```diff
@@ -353,3 +353,51 @@ An example ibc metadata file contains the following structure:
   ]
 }
 ```
+
+The following example alters the one above to go through an intermediate hop (cosmoshub), the file should be named `juno-cosmoshub-osmosis.json`. The intermediate hop needs to specify connection and client IDs for both chains it connects to:
```

Comment:
We may want to keep the files named by channel ends and avoid adding the intermediate chain. It could change and in general I doubt we would have more than one canonical/manually specified path between any two chains. Also we could have more than one intermediate chain so could get messy.

Reply:
Good point about the messiness with more than one intermediate chain, although I'm not sure how to deal with a couple of scenarios if we do that:
* testing multihop routes when a direct route is available (the two configs would conflict in the current format)
* more importantly upgrading from a multi-hop route to a direct route when one becomes available. One could think we can just replace the multi-hop route so there would be no conflict, but I can imagine there could be some resistance to that as people want to migrate to the new route over time and have the two co-exist for a while (before perhaps retiring the multi-hop route)

Reply:
I'm probably overthinking this, reverted to `<src>-<dst>.json` format